src/index.js

```diff
@@ -25,18 +36,55 @@ function calculateAll() {
 function removeProduct(event) {
   const target = event.currentTarget;
   console.log('The target in remove is:', target);
+  const deleteRow = target.parentNode.parentNode;
+  const initialParent = target.parentNode.parentNode.parentNode;
+  initialParent.removeChild(deleteRow);
```

Comment:
Another way to do this is using the method remove, like:

`target.parentNode.parentNode.remove();`